javascript/package-lock.json

```diff
@@ -20,9 +20,9 @@
       }
     },
     "node_modules/@grpc/grpc-js": {
-      "version": "1.7.3",
-      "resolved": "https://registry.npmjs.org/@grpc/grpc-js/-/grpc-js-1.7.3.tgz",
-      "integrity": "sha512-H9l79u4kJ2PVSxUNA08HMYAnUBLj9v6KjYQ7SQ71hOZcEXhShE/y5iQCesP8+6/Ik/7i2O0a10bPquIcYfufog==",
+      "version": "1.8.14",
+      "resolved": "https://momento-prod-401011790710.d.codeartifact.us-west-2.amazonaws.com/npm/npm-upstream/@grpc/grpc-js/-/grpc-js-1.8.14.tgz",
```

Comment:
Did you do a `mm dev login` that logged you in to our codeartifact repo?

You need to stash the changes to this file and remove your ~/.npmrc file and run npm install again so that all of the resolved urls in here point to registry.npmjs.org.

cc: @rishtigupta this is the thing I was telling you about yesterday - builds will fail on github if we accidentally end up with references to our codeartifact repo in our public/OSS projects.